app/views/publishers/new_auth_token.html.slim

```diff
@@ -1,13 +1,9 @@
 - content_for(:navbar_content) do
 
-.publisher-domain-panel.publisher-panel.col-center
-  .publisher-domain-name
-    | &nbsp;
```

Comment:
@lgebhardt I think this landed in an old commit from you- I've removed it b/c it was adding visual padding but I don't think it was doing anything besides that. Does that sound correct?